.github/workflows/print_github_metrics.py

```
@@ -0,0 +1,306 @@
+            metrics["created_at"] = created_at
+        
+        updated_at = workflow_data.get('updated_at')
+        if updated_at:
+            metrics["updated_at"] = updated_at
+        
+        # Duration and queue time
+        if created_at and updated_at:
+            try:
+                duration = (pd.to_datetime(updated_at) - pd.to_datetime(created_at)).total_seconds()
+                metrics["workflow_duration_seconds"] = duration
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)